src/components/shared/Snackbar/index.tsx

```diff
@@ -13,12 +21,20 @@ const Container = styled.View`
   border-radius: 10;
 `;
 
+const ActionContainer = styled.View`
+  display: flex;
+  align-items: center;
+  margin-left: auto;
+  padding-left: 16px;
+`;
+
 export interface SnackbarProps {
   text: string;
   testID?: string;
   show: boolean;
   setShow: (show: boolean) => void;
   timer?: Timer;
+  action?: React.ReactNode;
```

Comment:
I think only text, text color, and callback should be provided as action. 
Using ReactNode can burden users with implementation and cause break the struct.
How do you think?

Reply:
That would be better.
Thank you for a good idea.
I'll fix it right away.